module2/homework/transformContainers/transform.cpp

```diff
@@ -0,0 +1,18 @@
+#include "transform.hpp"
+
+#include <algorithm>
+#include <list>
```

Comment:
You have included list in the header. You do not need to include this one more time.